doc/modules/language-guide/pages/language-manual.adoc

```diff
@@ -1276,22 +1276,49 @@ The declaration `<dec>` of a `system` field must be a manifest `func` declaratio
 |  name | type | description
 
 | `heartbeat`  | `+() -> async ()+` | heartbeat action
+| `inspect_message` | `+{ caller : Principal; msg : <Variant>; arg : Blob } -> Bool+` | message predicate
```

Comment:
`inspect` sounds better to me.

Reply:
I could live with that. Let's see what the others say.